### ISSUE TITLE
Persistent Information Without Network Connection

### ISSUE THREAD

**Issue by thfondak** [scottisloud/WeatherBar#22]
App currently displays no information if internet access isn't available. Would be neat to have cached information available.

**Comment by scottisloud** [comment]
Good idea. Will try to see how to do the elegantly. The challenge will be in dealing with very old data, or instances where there was no cached data to begin with. But those are only challenges because I’m a novice, not because they’re technically insurmountable. 

May look into this for 0.0.5-a1 but I think it may be a few versions out.
